docs/source/python_api/mlflow.types.schema.rst

```diff
@@ -0,0 +1,6 @@
+mlflow.types.schema
```

Comment:
similar here for types.schema, seems like we shouldn't be introducing so many exposed modules for this PR

Reply:
Yeah I agree, I don't know how to make stuff visible to documentation without exporting the module though.

Reply:
Can we embed it into types? We should also get feedback from others on if we should release with these packages, but we can do that post-merge. For now let's just make sure it looks reasonable.

Reply:
done